### PR TITLE
test: rebaseline wheel tests according to new chromium

### LINKED ISSUE #7362: [Internal] page-drag tests fail on macOS 11.0

### PR DESCRIPTION
**Pull request by dgozman**
Fixes #7362.

### CODE REVIEW

File: tests/page/wheel.spec.ts [hunk 1]
```diff
@@ -20,15 +20,23 @@ it.skip(({ isElectron, browserMajorVersion, isAndroid }) => {
   // Old Electron has flaky wheel events.
   return (isElectron && browserMajorVersion <= 11) || isAndroid;
 });
+
+let deltaScale = 1;
+
+it.beforeAll(async ({ browserMajorVersion, browserName, platform }) => {
+  if (browserName === 'chromium' && browserMajorVersion >= 102 && platform === 'darwin')
```
Comment (rwoll) [inline tests/page/wheel.spec.ts:27]: Can you add a comment that explains why this divergence is needed (or link to relevant CR bits)?

**Review by rwoll — APPROVED** [review]

**Comment by pavelfeldman** [comment]
This goes against https://playwright.dev/docs/next/api/class-mouse#mouse-wheel

File: tests/page/wheel.spec.ts (hunk 1 above)
Comment (pavelfeldman) [reply tests/page/wheel.spec.ts:27]: I filed an issue for you: https://bugs.chromium.org/p/chromium/issues/detail?id=1324819
As long as scrollTop is good, we are Ok.

**Review by pavelfeldman — APPROVED** [review]

File: tests/page/wheel.spec.ts (hunk 1 above)
Comment (dgozman) [reply tests/page/wheel.spec.ts]: Thank you!

**Pull request merged**